x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.test.tsx

```diff
@@ -693,6 +685,7 @@ describe.skip('Response actions history', () => {
         'suspend-process',
         'processes',
         'get-file',
+        'execute',
```

Comment:
Should this be part of the PR? Asking because I know you've been working on 'execute' action for 8.8

Reply:
Yeah, this is needed cause of the latest changes on `main`. The backport PR should not include that commit though.